components/Article/Article.styled.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { css } from "styled-components";
+
+const StyledArticle = styled.article`
+  margin: 0 25px 25px 25px;
+
+  position: relative;
+  background-color: white;
+  border: none;
+  border-radius: var(--border-radius);
+`;
+
+const StyledDetailsLink = styled(Link)`
+  height: 35px;
+  position: absolute;
+  right: 10px;
+  bottom: 10px;
+  border-style: none;
+  color: var(--tertiary-color);
+`;
```

Comment:
You don't need `border-style: none` -  probably this is the holdover of a previous button styling 

```suggestion
const StyledDetailsLink = styled(Link)`
  height: 35px;
  position: absolute;
  right: 10px;
  bottom: 10px;
  color: var(--tertiary-color);
`;
```